src/components/camera-controller.js

```diff
@@ -0,0 +1,25 @@
+AFRAME.registerComponent("camera-controller", {
```

Comment:
This component actually isnt specific to camear at all, maybe chose another name.